app/decorators/email_context.rb

```diff
@@ -8,4 +8,8 @@ def initialize(user)
   def last_sign_in_email_address
     user.confirmed_email_addresses.order('last_sign_in_at DESC NULLS LAST').first
   end
+
+  def email_address_count
+    user.email_addresses.count
```

Comment:
If i add an email (unconfirmed) going by this count would allow me to delete a confirmed one.

Reply:
We are now checking for the last confirmed email address. See app/policies/email_policy.rb